source/DonutModel.h

```diff
@@ -41,6 +40,8 @@ class DonutModel {
 	cugl::Vec2 sgPos;
 	/** Angle of the donut in the world space */
 	float angle;
+	/** Size of the level */
+	float maxAngle;
```

Comment:
What's the reason behind making this a float? Do we plan on fractional degree level sizes?

Reply:
A lot of degree computation is done with floats, this is just to avoid having to cast all the time.